src/Entity/ShipmentServiceOptions.php

```diff
@@ -307,6 +350,33 @@ public function setNegotiatedRatesIndicator($NegotiatedRatesIndicator)
     /**
      * @return boolean
      */
+    public function isImportControlIndicator()
+    {
+        return isset($this->importControlIndicator);
```

Comment:
This will also return true when it is set to `false'. I know we don't check very well on the types of this indicators, but I think it will be best to check for `true` as the setter parameter seems to be boolean (see docblock below).